checker/tests/nullness/WeakHasherMapNullable.java

```diff
@@ -0,0 +1,20 @@
+import java.util.AbstractMap;
+import java.util.Map;
+import org.checkerframework.checker.initialization.qual.*;
+import org.checkerframework.checker.nullness.qual.*;
+import org.checkerframework.checker.regex.qual.*;
```

Comment:
Why import all these annotations, even though they aren't used?
Why not import `@Pure`, which is used?